neqo-transport/src/connection.rs

```diff
@@ -506,6 +506,12 @@ impl Connection {
                 Some(ref t) => {
                     qtrace!("TLS token {}", hex(&t));
                     let mut enc = Encoder::default();
+                    let rtt = self
+                        .loss_recovery
+                        .smoothed_rtt()
+                        .map(|v| u64::try_from(v.as_millis()).unwrap_or(0))
+                        .unwrap_or(0);
```

Comment:
```suggestion
                        .map(|v| u64::try_from(v.as_millis()))
                        .flatten()
                        .unwrap_or(0);
```
Maybe this is clearer?  Either works.

Reply:
I guess it will be reduced to 
```rust
let rtt = self.loss_recovery.rtt();
let rtt = u64::try_from(rtt.as_millis()).unwrap_or(0);
enc.encode_varint(rtt);
```
according to your comment https://github.com/mozilla/neqo/pull/519#discussion_r403774486